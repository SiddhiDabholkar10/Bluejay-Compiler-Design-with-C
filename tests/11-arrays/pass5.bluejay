fun f int()
  return 1;
arr[1] int := 1;
return arr[10*5+1+f()];
