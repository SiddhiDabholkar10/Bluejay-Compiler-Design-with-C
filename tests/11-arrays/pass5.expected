Function declared "f" returns int
Array written "arr" type int
Array read "arr" type int
